sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/optimizer/EliminateSortsSuite.scala

```diff
@@ -100,10 +104,10 @@ class EliminateSortsSuite extends AnalysisTest {
 
     val query = x.select($"a".as("x"), Year(CurrentDate()).as("y"), $"b")
```

Comment:
This test does not need to use `CurrentDate` at all. Let's change it to use `rand().as("y")`